WORKING_GROUPS.md

```diff
@@ -0,0 +1,207 @@
+
+Each Working Group has its own cadence, lifecycle and objectives.
+
+## Working Group Lifecycle
+
+**TODO: Define how a working group gets created, evaled and how to decide if its purpuse has been fulfilled and should be dismantled**
+
+## Current Working Groups
+
+### Development and Maintenance Working Groups
+
+**sub-index:**
+- [Release](#release)
+- [Specifications](#specifications)
+- [Files (Unixfs), DEX and Data Structures on top of IPLD](#files-unixfs-dex-and-data-structures-on-top-of-ipld)
+- [Runtime Support and Build](#runtime-support-and-build)
+- [QA, Testing and Dev Team Enablement](#qa-testing-and-dev-team-enablement)
+- [Security](#security)
+- [Performance and Benchmarking](#performance-and-benchmarking)
+
+#### Release
+
+The Release Working Group manages the releases for go-ipfs, js-ipfs and its internal dependencies.
+
+Responsibilities include:
+
+- Structure the released process.
+- Define the release Roadmap.
+- Decide what gets into each release.
+- Test releases.
+- Announce and gather feedback from the community on each release.
+
+#### Specifications
+
+The Specifications Working Group coordinates and participates on the specification writting process.
+
+Responsibilities include:
+
+- Ensure that specifications are up to date.
+- Enable developers and researches to contribute to the specification writting process with minimum overhead
+- That implementations can be compliant to the specs through formal testing (PDD and proofs).
+- Facilitate discussion around each spec making sure that proposals are followed up.
+
+#### Files (Unixfs), DEX and Data Structures on top of IPLD
+
+The Files, DEX and Data Structures on top of IPLD focuses on designed and improving the current Unixfs, MFS and Files API implementation, improving the versatility of Importers and Exporters (DEX) and exploring new ways of leveraging IPLD for data sets.
+
+Responsibilities include:
+
+- Design Importers and Exporters for different data sets
+- Maintain Unixfs
+- Improve the APIs that let users interact with Files on IPFS
+- Inform and guide users who want to define their own IPLD-based data formats
+
+#### Runtime Support and Build
+
+The Runtime Support and Build Working Group ensures that the implementations of IPFS are supported by multiple runtimes and architectures, while providing with the necessary set up to empower developers to make the best decisions while contributing to the project.
+
+Responsibilities include:
+
+- Create a Build pipeline for IPFS.
+- Create and monitor continuous testing for the multiple supported platforms (Architecture, OS'es, Browsrs).
+- Explore new runtimes and create Roadmaps towards supporting those.
+
+#### QA, Testing and Dev Team Enablement
```

Comment:
@VictorBjelkholm did you had the chance to review this PR? Let's sync up on what this means for 2018 quarter OKRs on _QA, Testing and Dev Team Enablement_ :)

Reply:
@diasdavid will review today.

Reply:
All seems good to me 👍